common/unsafe/src/main/java/org/apache/spark/unsafe/Platform.java

```diff
@@ -252,6 +272,7 @@ public static void throwException(Throwable t) {
       LONG_ARRAY_OFFSET = 0;
       FLOAT_ARRAY_OFFSET = 0;
       DOUBLE_ARRAY_OFFSET = 0;
+      _doubleAccessFunc = null;
```

Comment:
I am setting it to `null` to keep in line with the behavior of the `_UNSAFE` class variable. In case, `_UNSAFE` is null. In that case, any calls to the `Platform.get...` or `Platform.put...` will result in a `NullPointerException`. I'm assuming it is a programming error elsewhere to call these methods without checking ( [example](https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/memory/MemoryManager.scala#L200) ).